### PR TITLE
Improve Oracle types support

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Set;
+
+/**
+ * Oracle schema reader.
+ */
+public class OracleSchemaReader extends CommonSchemaReader {
+
+  public static final int INTERVAL_YM = -103;
```
Comment (jayfjin) [inline oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java:34]: what do these constants mean?

File: oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    if (ORACLE_TYPES.contains(sqlType)) {
+      switch (sqlType) {
+        case INTERVAL_DS:
+        case INTERVAL_YM:
+          return Schema.of(Schema.Type.STRING);
+        case TIMESTAMP_LTZ:
+        case TIMESTAMP_TZ:
+          return Schema.of(Schema.LogicalType.TIMESTAMP_MICROS);
+        default:
+          throw new SQLException(new UnsupportedTypeException(
```
Comment (jayfjin) [inline oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java]: is this code accessable based on the if condition you have above?

File: oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java (hunk 1 above)
Comment (ibarbashov) [reply oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java:34]: It is Oracle type constant, from Oracle JDBC Driver Implementation, and it doesn't match standard SQLTypes constants

File: oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java (hunk 2 above)
Comment (ibarbashov) [reply oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java]: yes, you right. Removed

File: oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java (hunk 1 above)
Comment (jayfjin) [reply oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java:34]: I see, would be good to have a comment about this

Comment (ibarbashov) [reply oracle-plugin/src/main/java/co/cask/oracle/OracleSchemaReader.java:34]: added

**Pull request merged**